src/util/format_expr.cpp

```diff
@@ -299,8 +299,9 @@ std::ostream &format_rec(std::ostream &os, const exprt &expr)
   else if(id == ID_if)
   {
     const auto &if_expr = to_if_expr(expr);
-    return os << format(if_expr.cond()) << '?' << format(if_expr.true_case())
-              << ':' << format(if_expr.false_case());
+    return os << "(" << format(if_expr.cond()) << " ? "
+              << format(if_expr.true_case()) << " : "
+              << format(if_expr.false_case()) << ")";
   }
```

Comment:
Use '(' etc. for the single letters?

Reply:
Done (plus two other similar changes in the same file)